nessie/lakectl_util.go

```diff
@@ -199,6 +201,11 @@ func normalizeCommitID(output string) string {
 	return string(s)
 }
 
+func normalizeShortCommitID(output string) string {
+	s := shortCommitIDRegExp.ReplaceAll([]byte(output), []byte("<COMMIT_ID_16>"))
```

Comment:
Relevant for other places, Go supply the 'string' variant - use: `ReplaceAllString`

Reply:
Same as for the previous one - Let's have it as a different PR, along with the refactoring to table-driven. Something like 'Lakectl Test Refactoring'. WDYT?

Reply:
Done